po/ru.po

```diff
@@ -0,0 +1,169 @@
+# Russian translation for spot package.
+# Copyright (C) 2021 THE spot'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the spot package.
+# Automatically generated, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: spot\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-03-16 19:27+0100\n"
+"PO-Revision-Date: 2021-03-16 19:21+0100\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: Paul Bragin <zeusmods@protonmail.com>\n"
+"Language: ru\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=ASCII\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

Comment:
According to [gettext documentation](https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html), Russian needs something like
```
Plural-Forms: nplurals=3; \
    plural=n%10==1 && n%100!=11 ? 0 : \
           n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;
```